planemo/lint.py

```diff
@@ -20,8 +23,28 @@ def build_lint_args(ctx, **kwds):
         skip = ctx.global_config.get("lint_skip", "")
         if isinstance(skip, list):
             skip = ",".join(skip)
-
     skip_types = [s.strip() for s in skip.split(",")]
+
+    for skip_file in kwds.get("skip_file", []):
+        with open(skip_file) as f:
+            for line in f.readlines():
+                line = line.strip()
+                if not line or line.startswith("#"):
+                    continue
+                skip_types.append(line)
+
+    from planemo.linters import (
+        biocontainer_registered,
+        conda_requirements,
+        doi,
+        urls,
+        xsd,
+    )
+    linters = list_linters([biocontainer_registered, conda_requirements, doi, urls, xsd])
+    for skip_type in skip_types:
+        if skip_type not in linters:
+            error(f"Unknown linter type {skip_type} in list of linters to be skipped. Known linters {linters}")
```

Comment:
This should do the trick. But

- the `list_linters` function will only be available with https://github.com/galaxyproject/galaxy/pull/17081, i.e. 24.0 any idea how to handle this
- the extra list passed as argument to `list_linters` is probably not necessary, since the extra linters need to be enables  explicitly on CLI (e.g. `--urls`) so it might not be useful to be able to skip them.

Also was wondering if its necessary to deal with the set of linter names changing with https://github.com/galaxyproject/galaxy/pull/17081 .. but this should influence only the `--skip...` parameters.